test/source/test.ts

```diff
@@ -26,12 +26,12 @@ export const internalTestState = { expectiIntentionalErrReport: false }; // upda
 process.setMaxListeners(30);
 
 const consts = { // higher concurrency can cause 429 google errs when composing
-  TIMEOUT_SHORT: minutes(1),
-  TIMEOUT_EACH_RETRY: minutes(3),
-  TIMEOUT_ALL_RETRIES: minutes(13), // this has to suffer waiting for semaphore between retries, thus almost the same as below
-  TIMEOUT_OVERALL: minutes(14),
-  ATTEMPTS: testGroup === 'STANDARD-GROUP' ? oneIfNotPooled(3) : process.argv.includes('--retry=false') ? 1 : 3,
-  POOL_SIZE: oneIfNotPooled(isMock ? 12 : 2),
+  TIMEOUT_SHORT: minutes(5),
+  TIMEOUT_EACH_RETRY: minutes(15),
+  TIMEOUT_ALL_RETRIES: minutes(65), // this has to suffer waiting for semaphore between retries, thus almost the same as below
+  TIMEOUT_OVERALL: minutes(70),
+  ATTEMPTS: testGroup === 'STANDARD-GROUP' ? oneIfNotPooled(6) : process.argv.includes('--retry=false') ? 2 : 6,
```

Comment:
Don't change the retry count, that will just increase the test time.

It's probably timing out due to too many failures, which it has to keep retrying.

Increasing the timeouts but bringing back the retry count has a better chance for the tests to finish.

Also maybe cron needs a new cert, I'll look into that separately.

Reply:
:+1: thanks!